src/redux/wallets/actions/pos-actions.ts

```diff
@@ -286,9 +274,29 @@ export const posAction = (
             },
             type
         );
-
         dispatch(setProcessTransactions(cloneDeep(txs)));
+    } catch (errorMessage) {
+        SentryCaptureException(new Error(JSON.stringify(errorMessage)));
+    }
+};
+
+export const signAndSendTransactions = (transactions: IBlockchainTransaction[]) => async (
+    dispatch: Dispatch<IAction<any>>,
+    getState: () => IReduxState
+) => {
+    const state = getState();
 
+    const appWallet = getSelectedWallet(state);
+    const account = getSelectedAccount(state); // Not sure its ok...
```

Comment:
you can do this, it's safe, or another option would be to get the account from the first tx from the list, I think it's safe to assume that all transactions are made from the same account.